c++/blocks.rst

```diff
@@ -65,15 +65,105 @@ Each project follows the same standard structure:
   |    |    |    |    +-- CMakeLists.txt
   |    +-- deps/
   
-This structure empowers consistency between the blocks published in biicode, it also enables working with different owner/blocks at the same time.
+This structure empowers consistency between the blocks published in biicode, it also enables working with *different owner/blocks* at the same time.
 
 .. container:: infonote
      
-     Tips naming your block:
+     See more info about working around with blocks:
+    
+     * :ref:`Working with your blocks<workflows_own_blocks>`
+     * :ref:`Working with any published block<workflows_any_block>`
+
+**Note** that you can add as many directories as you want inside a block, so you can create the ``include``, ``src``, ``test`` ones manually as well as *cpp files* in them.
+
+.. container:: infonote
+     
+     **Tips naming your block:**
+
         * Use underscores for multiple words
         * Use lowercase letters (Newbies will thank this. OS X and Windows have case-insensitive filesystems by default)
         * :ref:`Tag your STABLE versions <version_tags>`
 
+Block's web view
+----------------
+
+As you know, a block is where you place your code in biicode. Blocks published in the web are reused when biicode searchs for dependencies.
```

Comment:
The blocks available in the web are the ones you can depend on. 

** o algo así?